.github/workflows/build-wheel-macos-x86_64.yaml

```diff
@@ -359,6 +359,10 @@ jobs:
       run: |
         python${{ matrix.python_version }} -m pip install $GITHUB_WORKSPACE/dist/*.whl
 
+    - name: Install PennyLane-Lightning master
+      run: |
+        python${{ matrix.python_version }} -m pip install -i https://test.pypi.org/simple/ PennyLane-Lightning --pre --upgrade
+
```

Comment:
Let's add these to linux as well. And let's also add the specific commit here as well.